client/common/facebook_login.service.js

```diff
@@ -0,0 +1,74 @@
+var facebookLoginService = angular.module('facebookLoginService', []);
+
+facebookLoginService.factory('facebookLogin', facebookLogin);
+
+function facebookLogin($http, $q, $state) {
+    var service = {};
+    service.fbLoginSuccess = function(response) {
+        var def = $q.defer();
+        if (!response.authResponse) {
+            fbLoginError("Cannot find the authResponse");
+            return;
+        }
+        var authResponse = response.authResponse;
+        service.getFacebookProfileInfo(authResponse)
+            .then(function(profileInfo) {
+                console.log(profileInfo);
+                Session.set('FB_social_id', profileInfo.id);
```

Comment:
sir api for friend list for particular user needs these value .. and also we are not using ngStorage